src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolJudicialOfficeMappingSSCS.java

```diff
@@ -0,0 +1,633 @@
+package uk.gov.hmcts.reform.orgrolemapping.domain.service;
+
+import org.junit.runner.RunWith;
+import org.junit.jupiter.api.Test;
+import org.kie.api.runtime.rule.QueryResults;
+import org.kie.api.runtime.rule.QueryResultsRow;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.JudicialAccessProfile;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.RoleAssignment;
+import uk.gov.hmcts.reform.orgrolemapping.helper.TestDataBuilder;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import static org.hamcrest.collection.IsIterableContainingInAnyOrder.containsInAnyOrder;
+import static uk.gov.hmcts.reform.orgrolemapping.domain.service.RequestMappingService.ROLE_ASSIGNMENTS_RESULTS_KEY;
+
+
+@RunWith(MockitoJUnitRunner.class)
+class DroolJudicialOfficeMappingSSCS extends DroolBase {
+
+    @Test
+    void shouldReturnTribunalJudgeSalariedRoles() {
+
```

Comment:
write parametarised test by using @CSVParameter to verify similar functionalities

Reply:
Done